packages/manager/modules/nutanix/src/dashboard/general-info/translations/Messages_fr_FR.json

```diff
@@ -20,5 +23,8 @@
   "nutanix_dashboard_cluster_redeploy_modal_header": "Redéploiement du cluster",
   "nutanix_dashboard_cluster_redeploy_modal_description": "Il est nécessaire de désinstaller votre licence Nutanix de votre cluster dans Prism Central avant le redéploiement de votre cluster.",
   "nutanix_dashboard_cluster_redeploy_modal_continue": "Continuer",
-  "nutanix_dashboard_cluster_redeploy_modal_cancel": "Annuler"
+  "nutanix_dashboard_cluster_redeploy_modal_cancel": "Annuler",
+  "nutanix_dashboard_cluster_total_suscribed": "noeuds dans la souscription initale",
+  "nutanix_dashboard_cluster_node_to_deploy": "{{numberOfNodes}} noeud{{suffixPlurial}} à configurer",
+  "nutanix_dashboard_cluster_node_deployed": "{{numberOfNodes}} noeud{{suffixPlurial}} installé{{suffixPlurial}}"
```

Comment:
It doesn't work for all the languages, `s` is not always the suffix for plural. By the way, the variable should be `suffixPural`